### ISSUE TITLE
Requirements of existing directories stronger for GradleSite than Maven

### ISSUE THREAD

**Issue by russel** [breskeby/GradleSite#3]
I can do "mvn site" using Maven 3 on a project and it works fine.  Using GradleSite I get:

FAILURE: Build failed with an exception.
- What went wrong:
  A problem was found with the configuration of task ':site'.
  Cause: Directory '/home/users/russel/Repositories/Bazaar/Masters/ADS/src/site' specified for property 'sourceDir' does not exist.

I would have thought there would be a default in this situation.

**Comment by breskeby** [comment]
the plugin uses src/site as a default.

at the moment you should be able to change the input dir by adding

site{
    sourceDir = "src/customSiteDir/" as File
}

**Comment by russel** [comment]
Indeed, my problem is that Maven 3 builds the site even when src/site is not present, whereas GradleSite fails if the directory is not present.

**Comment by russel** [comment]
Also if src/site exists but src/site/apt or src/site/xdoc do not then execution fails with an exception but no explanation of what the problem is -- one has to infer they are directories that are required to exist for anything to happen.
